PhpAmqpLib/Wire/IO/StreamIO.php

```diff
@@ -224,10 +226,10 @@ public function read($len)
                 if ($this->canDispatchPcntlSignal) {
                     // prevent cpu from being consumed while waiting
                     if ($this->canSelectNull) {
```

Comment:
> Could you please be more specific in defining "network issues" you mentioned in #416
Updated the issue description, thank you. Replaced "network issues" with "connection is broken".